### ISSUE TITLE
Add portal 2 soundscape scripts

### ISSUE THREAD

**Issue by conneath** [Facepunch/garrysmod-requests#1896]
### Details
Portal 2 soundscape files are missing from gmod

**Comment by FlorianLeChat** [comment]
Can't you get them by mounting Portal 2 on Gmod?

**Comment by conneath** [comment]
> 
> 
> Can't you get them by mounting Portal 2 on Gmod?

nope, soundscape files arent mounted in gmod, they dont play in portal 2 maps
